### PR TITLE
Add new rewriter to spot easily what is not part of the translation file 

### PR DESCRIPTION
**Pull request by pfouque**
Looking for a tool to test my translation I found (remind) podebug with rewriter was a really good tool.
But It was missing an option to spot strings that have not been integrated yet.
I tried with Chef and Flipped, but it was still not enough obvious.

That's why I created the NSA rewriters 

Based on flipped it keeps variables but obfuscates all alphanumeric characters with a black square 

Perhaps it can serve the community, so I made this PR...

### CODE REVIEW

File: docs/commands/option_rewrite.rst [hunk 1]
```diff
@@ -143,3 +143,19 @@ messages are still readable.
    Run :doc:`pofilter` as a quick method to fix up incorrect changes, or
    upgrade to version 1.4.
 
+.. _option_rewrite#nsa:
+
+nsa
+=======
```
Comment (nijel) [inline docs/commands/option_rewrite.rst:149]: The underline should have same lenth as the text:

```suggestion
===
```

File: docs/commands/option_rewrite.rst [hunk 2]
```diff
@@ -143,3 +143,19 @@ messages are still readable.
...
+
+.. versionadded:: dev
```
Comment (nijel) [inline docs/commands/option_rewrite.rst:151]:
```suggestion
.. versionadded:: 3.7.3
```

File: docs/commands/podebug.rst [hunk 3]
```diff
@@ -58,7 +58,7 @@ Options:
 -S, --timestamp       skip conversion if the output file has newer timestamp
 -f FORMAT, --format=FORMAT     specify format string
 --rewrite=STYLE        the translation rewrite style: :doc:`xxx, en, blank,
-                       chef  (v1.2), unicode (v1.2) <option_rewrite>`
+                       chef  (v1.2), unicode (v1.2), nsa (dev) <option_rewrite>`
```
Comment (nijel) [inline docs/commands/podebug.rst:61]: This label is missing more styles, maybe use something more generic as label here would a better approach.

Comment (nijel) [reply docs/commands/podebug.rst:61]: It links to this: https://translate--4714.org.readthedocs.build/projects/translate-toolkit/en/4714/commands/option_rewrite.html

Comment (pfouque) [reply docs/commands/podebug.rst]: Thanks @nijel for the review! I renamed it "classified", and addressed the changes you pointed.

**Comment by nijel** [comment]
Merged, thanks for your contribution!

**Pull request merged**